src/helpers/mail.ts

```diff
@@ -18,6 +19,9 @@ export const sendMail = (payload: Payload): void => {
   }
 
   mail.sendMail(mailOptions, (err) => {
-    if (err) throw err
+    if (err) {
+      console.log(err.message);
```

Comment:
I prefer `console.error`

Reply:
output in cli same as `console.log` 
![image](https://user-images.githubusercontent.com/41193120/140670661-00cb486b-8670-448c-a7e8-62c02abe14c6.png)
![image](https://user-images.githubusercontent.com/41193120/140670665-f6f89f40-b307-466c-8b36-c07860210d97.png)